app/src/screens/NoWrappedTokens.js

```diff
@@ -65,25 +53,10 @@ const NoWrappedTokens = React.memo(function NoWrappedTokens({ isSyncing }) {
               height: 170px;
             `}
             src={emptyStateImg}
-            alt="No tokens here"
+            alt="No wrapped tokens yet"
           />
         }
       />
-      <Panel
```

Comment:
This isn't necessary; if we render the Panel at a higher level, using `wrapTokensPanel.requestOpen` will still be able to manipulate its open state.